app/styles/components/events/view/export.scss

```diff
@@ -0,0 +1,30 @@
+.string {
+  color: green;
+}
+
+.number {
+  color: darkorange;
+}
+
+.boolean {
+  color: blue;
+}
+
+.null {
+  color: magenta;
+}
+
+.key {
+  color: red;
+}
+
+.json.response {
+  max-height: 650px;
+  overflow: scroll;
+}
+
+.url {
+  overflow-y: hidden;
+  overflow-x: scroll;
```

Comment:
Remove overflow declarations. Pre should automatically take care of that for us.